app/script/components/list/conversationListCell.js

```diff
@@ -68,7 +68,7 @@ ko.components.register('conversation-list-cell', {
           </availability-state>
         <!-- /ko -->
         <!-- ko ifnot: conversation.is_one2one() && conversation.self.inTeam() -->
-          <span class="conversation-list-cell-name" data-bind="text: conversation.display_name(), css: {'text-theme': is_selected(conversation)}"></span>
+          <span class="conversation-list-cell-name" data-bind="text: conversation.display_name(), css: {'text-theme': is_selected(conversation), 'conversation-list-cell-name--bold': !!cell_state().description}"></span>
```

Comment:
Should we add a boolean value to the cell state instead of abusing the description?

Reply:
I think that would add unnecessary logic which would be needed only for this one line.